### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md [hunk 1]
```diff
@@ -20,3 +20,4 @@
 
 ## 1.0.0-preview.2
 
+- Official public preview of Azure.DigitalTwins.Core SDK
```
Comment (drwill-ms) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md:23]: Maybe a link to the readme and sample readme for more information about what is included?

File: sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md [hunk 2]
```diff
@@ -20,3 +20,4 @@
 
 ## 1.0.0-preview.2
```
Comment (drwill-ms) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md:21]: Shouldn't the newest releases be on top?

File: sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md (hunk 1 above)
Comment (drwill-ms) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md:23]: Could be under the Added header instead

File: sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md (hunk 2 above)
Comment (azabbasi) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md:21]: I think that would look better as well but every instance of this I have seen in the repo looks like this.

**Review by drwill-ms — APPROVED** [review]

**Pull request merged**